src/test/java/org/mockitousage/constructor/CreatingMocksWithConstructorTest.java

```diff
@@ -83,7 +147,7 @@ public void mocking_inner_classes_with_wrong_outer_instance() {
             fail();
         } catch (MockitoException e) {
             assertThat(e).hasMessage("Unable to create mock instance of type 'InnerClass'");
-            assertThat(e.getCause()).hasMessageContaining("Unable to find a matching 1-arg constructor for the outer instance.");
+            assertThat(e.getCause()).hasMessageContaining("Please ensure that the target class has a constructor that matches these argument types: [java.lang.String].");
```

Comment:
In this scenario we should inform the user somehow that the provided outer object is most likely incorrect because we cannot find matching constructor.